### PR TITLE
LESS to SASS, Change JS/CSS asset generation

### PR DESCRIPTION
**Pull request by ecobbold15**
Implements #46, #47 and #48

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,15 +31,16 @@
     "gulp-concat": "^2.6.0",
     "gulp-filter": "^3.0.1",
     "gulp-if": "^2.0.0",
+    "gulp-ignore": "^2.0.1",
     "gulp-imagemin": "^2.3.0",
     "gulp-jshint": "^1.11.2",
-    "gulp-less": "^3.0.3",
```
Comment (ecobbold15) [inline package.json]: Bye bye less

**Comment by ecobbold15** [comment]
The new bower stuff can be tested out by installing a few random extra bower packages and running the bower:js task, and then have a look in `static/dist/js/vendor/`.

**Comment by MrFirthy** [comment]
Where is Bootstrap sass being used? It's added in the `bower.json` file but I can't see where it is being used.

File: gulpfile.js [hunk 2]
```diff
@@ -40,56 +87,116 @@ var gulp = require('gulp'),
...
-});
+/* -----------------------------------------------------------------------------
+ * getLibsGlob
+ * Accepts: either a single or an array of lib collection names
+ * Returns: a glob pattern string
+ * Generates a glob pattern from the collection name of vendor libs, as defined
+ * in libs.vendor
+ * Can also accept an array of lib collection names - e.g. ['ie8', 'core']
+ * -------------------------------------------------------------------------- */
+function getLibsGlob (libCol) {
```
Comment (MrFirthy) [inline gulpfile.js]: This function is a thing of beauty

**Comment by MrFirthy** [comment]
Tested third party JS stuff independently and part of `core.min.js` and both work as expected. Will now merge this in. Bye bye LESS! :+1:

File: bower.json [hunk 3]
```diff
@@ -7,6 +7,7 @@
     "html5shiv": "~3.7.3",
     "jquery": "~2.1.4",
     "normalize-css": "~3.0.3",
-    "respond": "~1.4.2"
+    "respond": "~1.4.2",
+    "bootstrap-sass": "bootstrap-sass-official#~3.3.5"
```
Comment (ecobbold15) [inline bower.json]: Yeah don't worry about this, this one slipped through... it isn't being used for anything quite yet. I'm dealing with the whole BS3 switchover in a separate branch

Comment (MrFirthy) [inline bower.json]: No worries. It isn't doing any harm for now if it's going to be part of something else in the near future.

**Pull request merged**